web/src/main/java/org/springframework/security/web/server/authentication/X509AuthenticationWebFilter.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2002-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.web.server.authentication;
+
+import org.springframework.security.authentication.ReactiveAuthenticationManager;
+import org.springframework.security.web.authentication.preauth.x509.X509PrincipalExtractor;
+
+public class X509AuthenticationWebFilter extends AuthenticationWebFilter {
```

Comment:
Let's remove this as we don't extend AuthenticationWebFilter unless it is necessary (we prefer composition to inheritance)

Reply:
Thanks for your feedback, @rwinch ! I've removed the class now, although testing that filter was added is a bit tricky now and uses `TestReflectionUtils` to check `authenticationConverter` field. It is not ideal, but with a combination of integration test and convertes tests should provide good enough level of coverage.